src/lib/array.jl

```diff
@@ -10,10 +10,10 @@ using Base.Broadcast: broadcasted, broadcast_shape
 
 @adjoint Base.vect(xs...) = Base.vect(xs...), Δ -> (Δ...,)
 
-@adjoint copy(x::AbstractArray) = copy(x), ȳ -> (ȳ,)
+@adjoint copy(x::AbstractArray) = copy(x), ȳ -> (ȳ,)
```

Comment:
Not sure what's happening here, might be an editor settings thing. Either way it'd be great to remove these from the diff.

Reply:
Sorry about that, I tried to resolve a one-line conflict with github's editor and didn't notice. I'm hopeless at git but will see if I can un-do this.

Reply:
Well that was fun. I think now it's all cleaned up.